modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/sql/fun/SameFamilyOperandTypeChecker.java

```diff
@@ -55,7 +55,7 @@ public boolean checkOperandTypes(SqlCallBinding callBinding, boolean throwOnFail
 
     @Override
     public Consistency getConsistency() {
-        return Consistency.LEAST_RESTRICTIVE;
+        return Consistency.NONE;
```

Comment:
seems now it`s identical to super method and can be removed

Reply:
default implementation is not under our control, thus explicit declaration seems to be safer